INSTALLATION.md

```
@@ -380,3 +390,72 @@ If you don't want this hook to run, you can manually opt out of this using the `
+### Prerequisites
+
+- Firstly, You should have locally setup the Talawa-Admin repo using [Setting up this repository](#setting-up-this-repository)
+- Ensure **Node.js** and **npm** are correctly installed and available for the specified user and group.
+- It’s recommended to use **nvm** (Node Version Manager) for better management of different Node.js versions.
+
+---
+
+### Service Configuration
+
+#### 1. **Copy the `talawa_admin.service` file**
+   - Place the `talawa_admin.service` file in the appropriate systemd directory based on your Linux distribution:
```

Markdownlint (0.37.0)

Expected: 0; Actual: 3
Unordered list indentation

(MD007, ul-indent)